src/site/content/en/blog/carousel-best-practices/index.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: Best practices for carousels
+subhead: Optimize carousels for performance and usability.
+authors:
+  - katiehempenius
+description:  Learn how to optimize carousels for performance and usability.
+date: 2021-01-26
+hero: hero.jpeg
+tags:
+  - blog
+  - fast
+  - performance
+  - web-vitals
+  - internationalization
+---
+
+A carousel is a UX component that displays content in slideshow-like manner. Carousels can "autoplay" or be navigated manually by users. Although carousels can be used elsewhere, they are most frequently used to display images, products, and promotions on homepages.
+
```

Comment:
I think it would be good to include a visual example here. The explanation is good but a picture is quicker to process. Every dev has seen a carousel but not everyone calls them that ("slider" is also common and we have a huge non-native English speaker audience).